packages/paypal-fastlane-integration/src/is-paypal-fastlane-address.ts

```diff
@@ -28,9 +28,14 @@ const isEqualToPayPalFastlaneAddress = (
     paypalFastlaneAddress: Partial<ComparableAddress>,
 ): boolean => isEqual(normalizeAddress(bcAddress), normalizeAddress(paypalFastlaneAddress));
 
-const isPayPalFastlaneAddress = (address: Address, addresses: Address[]): boolean =>
-    addresses.some((paypalFastlaneAddress) =>
+const isPayPalFastlaneAddress = (address: Address, addresses?: Address[]): boolean => {
+    if (!addresses || addresses.length === 0) {
```

Comment:
`if (!addresses || !addresses.length)`

Reply:
`addresses.length === 0` looks more recognizable to me, so I can easily detect why I need this condition. I prefer to keep it as it is, if you don't mind